### DISCUSSION TITLE
The vector tiles of arcgis cannot be rendered

### DISCUSSION THREAD

**Discussion by qq908323236** [mapsforge/vtm#1017] [Q&A] [answered]
tile url ：https://basemaps.arcgis.com/arcgis/rest/services/OpenStreetMap_v2/VectorTileServer/tile
tile path ：/{Z}/{Y}/{X}.pbf

Modified OpenMapTilesMvtTileSource
![image](https://user-images.githubusercontent.com/19360225/218395435-7118d846-a850-4425-9b02-156c2fc05e20.png)

![image](https://user-images.githubusercontent.com/19360225/218395490-0d8ceb8d-9672-49bd-9fa4-751e743e60a7.png)

nothing is showing on the map

**Comment by devemux86** [comment, marked as answer]
It may need writing a [render theme](https://github.com/mapsforge/vtm/blob/master/vtm-themes/resources/assets/vtm/openmaptiles.xml), if it uses custom tags.
OpenMapTiles theme layers are described [here](https://openmaptiles.org/schema/).

Or another [TileDecoder](https://github.com/mapsforge/vtm/blob/master/vtm-mvt/src/org/oscim/tiling/source/mvt/TileDecoder.java), if vector tiles have special format.
